examples/create-image-dataset.py

```diff
@@ -0,0 +1,37 @@
+"""
+Creating an Image Dataset from Local PNGs/JPGs
+
+There are two ways two create a data of PNGs/JPGs depending on whether the images are
```

Comment:
In the near future there will be an additional way which will be to upload the image files directly as is possible to upload pdfs. This means we don't need to create a temporary csv using the client